package.json

```diff
@@ -28,22 +28,22 @@
     "server.js"
   ],
   "dependencies": {
-    "http-proxy": "1.11.1",
+    "http-proxy": "^1.18.1",
     "proxy-from-env": "0.0.1"
   },
   "devDependencies": {
-    "coveralls": "^2.11.6",
+    "coveralls": "^3.1.0",
     "eslint": "^2.2.0",
-    "istanbul": "^0.4.2",
     "lolex": "^1.5.0",
-    "mocha": "^3.4.2",
-    "nock": "^8.2.1",
-    "supertest": "^2.0.1"
+    "mocha": "^9.0.1",
+    "nock": "^13.1.0",
+    "nyc": "^15.1.0",
+    "supertest": "^6.1.3"
   },
   "scripts": {
     "lint": "eslint .",
     "test": "mocha ./test/test*.js --reporter spec",
-    "test-coverage": "istanbul cover ./node_modules/.bin/_mocha -- test/test.js test/test-ratelimit.js --reporter spec"
+    "test-coverage": "nyc npm run test"
```

Comment:
This is different from before - previously I explicitly listed the tests of interest to exclude `test-memory.js`, which basically runs a simple test repeatedly.

Reply:
Now I understand. I’ve correct that.